databricks/koalas/series.py

```diff
@@ -1237,6 +1237,50 @@ def to_list(self):
 
     tolist = to_list
 
+    def drop_duplicates(self, inplace=False):
+        """
+        Return Series with duplicate values removed.
+
+        Parameters
+        ----------
+        inplace: bool, default False
+            If True, performs operation inpalce and returns None.
+
+        Returns
+        -------
+        Series
+            Series with deplicates dropped.
+
+        Examples
+        --------
+        Generate a Series with duplicated entries.
+        >>> s = ks.Series(['lama', 'cow', 'lama', 'beetle', 'lama', 'hippo'],
+        ...               name='animal')
+        >>> s
+        0      lama
+        1       cow
+        2      lama
+        3    beetle
+        4      lama
+        5     hippo
+        Name: animal, dtype: object
+
+        >>> s.drop_duplicates()
+        1       cow
+        0      lama
+        5     hippo
+        3    beetle
+        Name: animal, dtype: object
+        """
+        kdf = self._kdf.drop_duplicates()
```

Comment:
`self.to_frame().drop_duplicates()`? 
`self._kdf` doesn't know the current `scol`. Also `self._kdf` might have other columns, then the result could differ.
E.g., consider something like `(s + 1).drop_duplicates()`.

Reply:
ehh, thanks! didn't notice this effect. 👍